ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -251,6 +251,11 @@ def test_update_simple(self, host):
                         valid_not_after=fuzzy_date,
                     ))
         host.retrieve()
+        # test host-find with --certificate
+        command = host.make_find_command(
+            fqdn=u'%s' % host.fqdn, usercertificate=host_cert)
```

Comment:
Why do you use ``u'%s' % host.fqdn`` instead of simple ``host.fqdn``?

Reply:
you are right, no need to use this as host.fqdn is already built with `self.fqdn = u'%s.%s' % (name, self.api.env.domain)` in HostTracker.